JohnnySonTrinh/remindicare
# Initial set up

- Create a basic Django Project
- Add Home App
- Add Basic Templates

Fixes https://github.com/JohnnySonTrinh/remindicare/issues/26

- [x ] New feature (non-breaking change which adds functionality)


## How To Test This PR?

If in gitpod, to install run: `pip3 install -r requirements.txt`
To run the project: `python3 manage.py runserver`

If developing in VsCode:
Create venv locally (if you haven't already done so): `python -m venv .venv`
Activate the venv:

- Windows: `.\.venv\Scripts\activate`
- Mac: source `.venv/bin/activate`

Run server: `python3 manage.py runserver`

Open the server and the basic template with 'Hello World' should show.
